frontend/src/modules/profile/view.jsx

```diff
@@ -162,7 +162,10 @@ const ProfileView = ({ ...props }) => {
         );
       })();
     }
-  }, [profile, pendingItems, reviewItems, reviewedItems, stakeholdersData]);
+    // NOTE: dependency needs to be []. Ignore the linter warning, because
```

Comment:
I don't understand the comment, now. We have a dependency on `profile`, but the comment says `[ ]`. The comment could explicitly say we don't want to depend on the other items, and we still need to depend on `profile`?

Reply:
Yep, that comment should be fixed. We still need the profile one.